apps/challenges/views.py

```diff
@@ -19,6 +21,7 @@
 from .serializers import ChallengeSerializer, ChallengePhaseSerializer
 
 
+@throttle_classes([UserRateThrottle])
```

Comment:
can you please move this below `authentication_classes`. Idea is to follow alphabetically write down decorators. `authentication_classes` is definitely not in the order, but lets not do it for others.

Also can you make sure of this point for each and every place where `throttle_classes` decorator is used.